packages/react/src/components/Table/TableBody/TableBodyLoadMoreRow/TableBodyLoadMoreRow.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DataTable } from 'carbon-components-react';
+import { ArrowDown16 } from '@carbon/icons-react';
+
+import { settings } from '../../../../constants/Settings';
+
+const { TableRow, TableCell } = DataTable;
+const { iotPrefix } = settings;
+
+const propTypes = {
+  /** The unique row id */
+  id: PropTypes.string.isRequired,
+  /** The unique id for the table */
+  tableId: PropTypes.string.isRequired,
+  /** The unique id for the table test */
+  testId: PropTypes.string.isRequired,
+  /**  total columns has the overall total */
+  totalColumns: PropTypes.number.isRequired,
+  /** call back function for when load more row is clicked */
+  onRowLoadMore: PropTypes.func.isRequired,
```

Comment:
what are the params?

Reply:
updated the comment with `call back function for when load more row is clicked  (rowId) => {} `